content/en/docs/release-oversight/troubleshooting-payload-failures.md

```diff
@@ -28,7 +28,15 @@ tools to check for larger trends related to tests, errors and events.
 - [CI Search](https://search.ci.openshift.org/) - If you have found a particular log entry or test failure and want to search bugs and other junit runs to find:
     - How frequently does it occur
     - Did it start or spike recently
-    - Is there an open bug against it <br><br>
+    - Is there an open bug against it 
+    - [CI Search Metrics](https://search.ci.openshift.org/graph/metrics)
+      - Are you hitting timeouts and want to see if the duration of the job has increased
+        - [job:duration:total:seconds](https://search.ci.openshift.org/graph/metrics?metric=job%3Aduration%3Atotal%3Aseconds&job=periodic-ci-openshift-release-master-ci-4.11-e2e-gcp-upgrade)
+      - Is there a sudden increase in api errors
+        - [cluster:api:errors:total:requests](https://search.ci.openshift.org/graph/metrics?metric=cluster%3Aapi%3Aerrors%3Atotal%3Arequests&job=periodic-ci-openshift-release-master-ci-4.11-e2e-gcp-upgrade)
+      - Or other metrics <br><br>
+
+metrics
```

Comment:
I think you wanted "Metrics" to be a section heading?

Reply:
I think it was a place holder and I did a poor job reviewing before pushing...